### ISSUE TITLE
[Quest][Northrend] The last line of defense - Evading issue

### ISSUE THREAD

**Issue by Kepithor** [dalaranwow/dalaran-wow#5386]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The quest "The last line of defense" in icecrown is bugged. The frostbroods keep evading so cannot be killed.**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. accept quest
2. attempt quest
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by pegesus328** [comment]
You guys say closed, but the problem still persists?

**Comment by Rushor** [comment]
The issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rahizm** [comment]
Some more information with this quest, nerubians will run around, but evade bug when hit. If hit again once they get back to their starting area, they take damage. The dragons will evade bug indefinitely, however if they have something to attack like a player, pet, etc. they can be damaged by the cannons. The workaround right now is to group up and have one person bait a dragon while another person shoots it. This seems to be a problem in the pathing and targeting behaviors. The second major problem is that players using the cannons will register as falling when they get out. They are able to get in and out of the cannon, but can not move, mount, hearth, use the unstuck tool, or teleport and simply show the falling animation while standing on the cannon tower.

**Comment by thestranger064** [comment]
I am writing as of today, this is not fixed yet

Also the Link http://www.dalaran-wow.com/news/  is A dead page btw.

Now According to http://www.dalaran-wow.com/changelog/21/the-path-to-a-new-dawn
 The Last Line Of Defense: **Behaviour improvements to Frostbrood Destroyer.**

Not working at all.
heres a few screeen shots
https://imgur.com/a/G35y3vd

**Comment by Dancingirl** [comment]
Why is this closed? We are may 19th and it is still a problem! There are no other quests available if we don't complete this one...

**Comment by Real-Scaramouche** [comment]
It's July 8th and this quest is still bugged so maybe reopen this issue or let someone else create a new thread and keep it open rather than putting a reference here with zero information whatsoever.

**Comment by Chenocide** [comment]
Gonna quote some stuff to you since you seem to have a hard time reading.

> The issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR).

> This means that the issue exists in game at the moment i wrote this message.

> Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game 
> server.

It's not on live but is fixed on the PTR. Opening a new thread will be closed right away.

**Comment by thestranger064** [comment]
Gonna quote some stuff to you since you seem to have a hard time reading.

The issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR).

This means that the issue exists in game at the moment i wrote this message.

Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game
server.

It's not on live but is fixed on the PTR. Opening a new thread will be closed right away.

Except we know this, - if you read my comment it says thats its been fixed on changelog - thus why we are wondering why this is closed - and why its in the changelog in the firstplace.

**Comment by Chenocide** [comment]
At least half of the changelog hasn't been ported yet and won't be until Ruby Sanctum is out. You're just gonna have to wait like the rest of us.
